tilequeue/format/__init__.py

```diff
@@ -102,14 +119,22 @@ def format_vtm(fp, feature_layers, zoom, bounds_merc, bounds_lnglat):
 # package of tiles as a metatile zip
 zip_format = OutputFormat('ZIP Metatile', 'zip', 'application/zip',
                           None, None, None)
+# MVT, but written out by coanacatl/wagyu
+coanacatl_format = OutputFormat('MVT/Coanacatl', 'mvt',
+                                'application/x-protobuf', format_coanacatl,
+                                5, supports_shapely_geom)
 
 extension_to_format = dict(
     json=json_format,
     topojson=topojson_format,
     vtm=vtm_format,
     mvt=mvt_format,
     mvtb=mvtb_format,
-    zip=zip_format
+    zip=zip_format,
+    # NOTE: this isn't actually the extension of the format; coanacatl writes
+    # files ending '.mvt'. this is just to give us something to call this
+    # format in config files.
+    coanacatl=coanacatl_format,
```

Comment:
Do we need a paired change in RAWR tiles config?

Reply:
No, it's only the tilequeue config that needs to change to support this.